documentation/versioned_docs/version-1.8.x/configuration/environments_file.md

```diff
@@ -25,6 +25,8 @@ bar:
     - env-token-name: "BAR_TOKEN_ENV_VAR"
 ```
 
+==env-token-name is the name of the env variable which holds your token. ex: `export BAR_TOKEN_ENV_VAR=XXXXXXXXX==`
```

Comment:
Thanks a lot for your contribution @fwyattblake ! 

Should those `==` be around that sentence? 

Also please kindly sign the CLA so we can merge this change